docs/getting_started/install.rst

```diff
@@ -0,0 +1,101 @@
+as they provide a way to seamlessly switch between different projects without conflicts.
+
+Using Conda environments is not mandatory to be able to install and use ProteusLib; however, it is strongly recommended.
+
+To use Conda environments, the ``conda`` package manager is required. Refer to the `Conda installation guide <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`_ for detailed steps on how to install Conda for your operating system.
+
+General installation
+--------------------
+
+If you are going to use ProteusLib's functionality, but *do not* plan to contribute to ProteusLib's codebase, choose this option.
+
+#. Create a Conda environment (in this example, named ``proteuslib``) where ProteusLib and its runtime dependencies will be installed:
+
+	.. code-block:: shell
+
+		conda create --name proteuslib --yes python=3.8
+
+#. Activate the ``proteuslib`` environment:
+
+	.. code-block:: shell
+
+		conda activate proteuslib
+	
+	To verify that the correct environment is active, run the following command:
+
+	.. code-block:: shell
+
+		python -c "import sys; print(sys.executable)"
+	
+	If the environment was activated correctly, its name should be contained in the path displayed by the above command.
+
+	.. important:: The ``conda activate`` command described above must be run each time a new terminal/console session is started.
+
+#. Install ProteusLib using ``pip``:
+
+	.. code-block:: shell
+
+		pip install https://github.com/nawi-hub/proteuslib/archive/main.zip
+
+#. To verify that the installation was successful, try running the ProteusLib test suite using ``pytest``:
+
+	.. code-block:: shell
+
+		pytest
+
+For ProteusLib developers
+-------------------------
+
+If you plan to contribute to ProteusLib's codebase, choose this option.
+
+.. note:: Typically, *contributing to ProteusLib* will involve opening a Pull Request (PR) in ProteusLib's repository. For more information, refer to :ref:`developer-guide`.
+
+#. Create a Conda environment (in this example, named ``proteuslib-dev``) where ProteusLib and all dependendencies needed for development will be installed, then activate it:
+
+	.. code-block:: shell
+
+		conda create --name proteuslib-dev --yes python=3.8 && conda activate proteuslib-dev
+
+	.. note:: For more information about using Conda environments, refer to the ":ref:`about-conda`" section above.
+
+#. Clone the ProteusLib repository to your local development machine using ``git clone``, then enter the newly created ``proteuslib`` subdirectory:
+
+	.. code-block:: shell
+
+		git clone https://github.com/nawi-hub/proteuslib && cd proteuslib
+
+#. Install ProteusLib and the development dependencies using ``pip`` and the ``requirements-dev.txt`` file:
+
+	.. code-block:: shell
+
+		pip install -r requirements-dev.txt
```

Comment:
Shouldn't we also tell developers to use the `-e` flag?

Reply:
That's a good point. The `-e/--editable` flag is, by design, built into `requirements-dev.txt`, with the underlying idea that `pip install -r requirements-dev.txt` is the single, consistent command to run to set up the developer environment.

It could be worth to note this here. Perhaps using a `.. note::` directive pointing to the (currently empty) developer how-to guide?

Reply:
I had forgotten (and didn't check) that it was built in to `requirements-dev.txt`. I'm happy with this as-it-is then.